src/lib/nvme_pt_ioctl.c

```diff
@@ -1728,10 +1724,9 @@ int opal_add_usr_to_lr_pt(struct sed_device *dev, const char *key, uint8_t key_l
 	struct sed_key disk_key;
 	int ret = 0;
 	struct opal_device *opal_dev;
-	uint32_t who;
+	uint32_t who, lk_type = lock_type;
```

Comment:
why do we need lk_type variable? can we reuse lock_type variable ?

Reply:
The reason for assigning it to a uint32_t type is that we needn't have to check for the negative value being passed. Only one check condition is sufficient.

Reply:
Ok, make sense